README.md

```diff
@@ -87,6 +88,21 @@ Beside every option from the [interfaces](#interfaces) dict, the following optio
 | ___key___   | string |         | First port or interface in between the link should be made   |     Y    |
 | ___value___ | string |         | Second port or interfaces in between the link should be made |     Y    |
 
+### network_management_routes
+| Option      | Type                   | Default | Description                                     | Required |
+|-------------|------------------------|---------|-------------------------------------------------|:--------:|
+| ___key___   | string                 |         | IP address or IP network as CIDR                |     N    |
+| ___value___ | [dict](#route_options) |         | Route option to set for specified network or IP |     Y    |
+
+#### route_options
+| Option    | Type    | Default                               | Description                                | Required |
+|-----------|---------|---------------------------------------|--------------------------------------------|:--------:|
+| gateway   | string  |                                       | IP address to use as gateway               |     N    |
+| interface | string  |                                       | Interface/device to us to forward traffic  |     N    |
+| source    | string  |                                       | Source IP address to us to forward traffic |     N    |
+| metric    | integer |                                       | Metric to use for this route definition    |     N    |
+| mtu       | integer | `{{ network_management_default_mtu}}` | MTU to use for this route                  |     N    |
```

Comment:
space
Also, out of interest, what does the MTU setting effect, will it shrink packets to the requested MTU or only send ones small enough through this route?

Reply:
It will set a mtu for the route, this is done by iproute2.
So it should send all packages with a mtu of X over this route, even if the interface is configured with a mtu of X+N.